modules/check_in/app/services/travel_claim/client.rb

```diff
@@ -12,9 +12,9 @@ class Client
     CLAIMANT_ID_TYPE = 'icn'
     TRIP_TYPE = 'RoundTrip'
 
-    attr_reader :settings, :check_in
+    attr_reader :settings, :check_in, :client_id
```

Comment:
Ticket says "We need to use a different client number for submitting claims for Oracle Health sites."; 

Could you please clarify if we need to use a different `client_id` or `client_number` for oracle healthcare?

Reply:
You're right, I confused `client_id` with `client_number`. `client_number` is what we need to parameterize. corrected in a62248e51d9616f707877e368678eb4cde17b915